lib/features/base/base_mailbox_controller.dart

```diff
@@ -442,6 +444,51 @@ abstract class BaseMailboxController extends BaseController {
     }
   }
 
+  void openConfirmationDialogSubaddressingAction(
+      BuildContext context,
+      ResponsiveUtils responsiveUtils,
+      ImagePaths imagePaths,
+      MailboxId mailboxId,
+      String mailboxName,
+      String subaddress,
+      Map<String, List<String>?>? currentRights, {
+        required Function(MailboxId, Map<String, List<String>?>?, MailboxActions) onAllowAction
```

Comment:
Should define name function for it to more clearly

Reply:
Oh no, I mean: use `typedef` to define a function type, which makes the code more concise and readable. E.g:
```
typedef FunctionName = ReturnType Function(ParameterType1, ParameterType2, ...);
```